packages/scripts.nix

```diff
@@ -16,6 +16,83 @@
     text = builtins.readFile ./destroy-coco-aks.sh;
   };
 
+  upload-image = writeShellApplication {
+    name = "upload-image";
+    runtimeInputs = with pkgs; [
+      azure-cli
+      gnused
+      uplosi
+    ];
+    text =
+      let
+        image = pkgs.image-podvm;
+      in
+      ''
+        set -euo pipefail
+
+        subscriptionId=""
+        location="GermanyWestCentral"
```

Comment:
we should add location as var to the terraform to prevent location mismatch between cluster and image location.